### ISSUE TITLE
Please update new version of Laravel 5.3

### ISSUE THREAD

**Issue by Reegan01** [JesseObrien/laravel-cheatsheet#146]
Please update the latest code snippts in your code

**Comment by Dragas** [comment]
see https://github.com/mclxly/laravel5-cheatsheet

**Comment by Reegan01** [comment]
@Dragas Yeah, the page have Laravel 5.3 cheat sheet But I can't able to download the PDF version. When I try to download it throw "400: Invalid request".

**Comment by Dragas** [comment]
You can still save the page.
![a](http://i.imgur.com/TRLwnPL.png)

**Comment by JesseObrien** [comment]
I would love to update this to 5.x releases, but I don't have a ton of time to do so right now. I'm working on seeing if anyone I know wants to maintain this or take on that task of updating it. For now, the fork (https://github.com/mclxly/laravel5-cheatsheet) is pretty decent. There's definitely some work and things I would like to change about it though.
